cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/BatchElasticsearchSink.java

```diff
@@ -50,7 +50,7 @@
 @Description("CDAP Elasticsearch Batch Sink takes the structured record from the input source" +
   " and converts it to a json, then indexes it in elasticsearch using the index, type, and id specified by the user." +
```

Comment:
Put the spaces at the end of the strings rather than the beginning. This string needs a space at the end, and so does the previous one.